portfolio/styles/RandomReceiptWithLabels.module.css

```
@@ -0,0 +1,462 @@
+  font-size: 1.5rem;
+  font-weight: 600;
+  color: var(--text-color);
+}
+
+.labelList {
+  margin: 1rem 0 1rem !important;
+  padding-left: 1.5rem !important;
+  list-style-type: disc !important;
+  list-style-position: outside !important;
+  display: block !important;
+  list-style: disc outside !important;
```

Biome (2.1.2)

[error] Unexpected shorthand property list-style after list-style-type



(lint/suspicious/noShorthandPropertyOverrides)